src/pages/Books.js

```diff
@@ -1,20 +1,24 @@
-import React from 'react';
-import { useSelector } from 'react-redux';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 import Form from '../components/Form';
 import Book from '../components/Book';
+import { fetchBookApi } from '../redux/books/books';
 
 const BookList = () => {
   const books = useSelector((state) => state.books);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchBookApi());
+  }, [dispatch]);
   return (
     <div>
       <h2>List of Books</h2>
       <ul>
         {books.map((book) => (
           <Book
-            key={book.id}
-            author={book.author}
+            key={book.item_id}
             title={book.title}
-            id={book.id}
+            item_id={book.item_id}
             category={book.category}
```

Comment:
- [ ] Good job on  adding, deleting, and fetching your books to the API but when rendering your books there is a bit of issue, When I go to the categories page it works fine I see the message `under construction but when I navigate back to books they duplicate themselves but if I reload the page the duplication disappears, Kindly have a look at that video and fix the issue :point_down: 

https://user-images.githubusercontent.com/75126481/149944326-60b98c4f-8373-4eba-830b-be2c0373cb1c.MP4

Reply:
Thank you very much for the observation. The book reducer was returning a merged array previously

`return [...state, action.payload];`

Reply:
You are welcome @Gbengacode